bedrock/settings/base.py

```diff
@@ -689,6 +689,20 @@ def facebook_tab_url_lazy():
 # Optimize.ly project code
 OPTIMIZELY_PROJECT_ID = None
 
+# Fx Accounts iframe source
+# Prod, stage, dev, & demos have this set in their local.py files.
+#
+# For local testing:
+#   - Point FXA_IFRAME_SRC to stomlinson's instance in your own local.py:
+#     FXA_IFRAME_SRC = 'https://stomlinson.dev.lcip.org/'
```

Comment:
Are we going to get a nicer way to test locally in the long run? Seems like depending on stomlinson's own dev instance is probably not what we want long-term?